.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+jobs:
+  release:
+    executor:
+      docker:
+        - image: alpine:latest
```

Comment:
Using circleci is not going to be a win unless you use our image-builder base image / orb.